### DISCUSSION TITLE
local Superset with Helm

### DISCUSSION THREAD

**Discussion by markusf1895** [apache/superset#27575] [General]
### Bug description

On Macbook Pro with M1:

I run into a timeout if running: `helm upgrade --install --values my-values.yaml superset superset/superset`

Please help.

### How to reproduce the bug

I did:
1. `helm repo add superset https://apache.github.io/superset` => `"superset" has been added to your repositories`
optional: `helm search repo superset` => `superset/superset, Chart Version: 0.12.6, App Version: 3.1.0`
2. `kind create cluster --name superset`
3. add `my-values.yaml` from here: https://github.com/apache/superset/blob/master/helm/superset/values.yaml
4. `helm upgrade --install --values my-values.yaml superset superset/superset` => `Release "superset" does not exist. Installing it now.`
5. After some time I get `Error: UPGRADE FAILED: post-upgrade hooks failed: 1 error occurred: * timed out waiting for the condition`

### Superset version

master / latest-dev

### Python version

I don't know

### Node version

I don't know

### Browser

Chrome

### Checklist

- [ ] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [ ] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by lodu** [comment]
I have this on a k3s installed on Debian 12.2 (amd64) as well.

First time using k8s/k3s so I thought I was doing something wrong.

EDIT: it was a bad config for me, try adding `--debug` flag to the helm upgrade command.

**Comment by michael-s-molina** [comment]
Moving this to a GitHub discussion given:
> 🙏 Asking a question or getting help: post in the [Superset Slack chat](http://bit.ly/join-superset-slack) or [GitHub Discussions](https://github.com/apache/superset/discussions) under "Q&A / Help".
